pages/common/git-lfs.md

```diff
@@ -0,0 +1,24 @@
+# git lfs
+
+> Work with large files in Git repositories.
+> More information: <>.
```

Comment:
Looks like you missed out the URL. :wink:
```suggestion
> More information: <https://git-lfs.github.com>.
```

Reply:
Gosh, looks like I did!